.github/workflows/pull-request.yml

```diff
@@ -128,6 +128,7 @@ jobs:
     needs:
       - analyze-changes
     uses: ./.github/workflows/ci.yml
+    secrets: inherit
```

Comment:
Secrets don't get passed by default. This was why our build times have balloned.